packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -72,6 +75,8 @@ const Trade = observer(() => {
     const [category, setCategory] = React.useState(null);
     const [subcategory, setSubcategory] = React.useState(null);
     const [swipe_index, setSwipeIndex] = React.useState(0);
+    const [open_launch_modal, setOpenLaunchModal] = React.useState(true);
+
```

Comment:
error  'getDecimalPlaces' is defined but never used      no-unused-vars
error  'has_barrier' is assigned a value but never used  no-unused-vars

Reply:
fixed